client/jest.config.js

```diff
@@ -21,8 +21,7 @@ module.exports = {
   collectCoverage: true,
 
   // An array of glob patterns indicating a set of files for which coverage information should be collected
-  // collectCoverageFrom: null,
-
+  collectCoverageFrom: ["**/*.{js,jsx}", "!**/node_modules/**"],
```

Comment:
Should this file be added to `.eslintignore`?

Reply:
I ended up just fixing that line as it looks like the rest of the file following the lint rules.